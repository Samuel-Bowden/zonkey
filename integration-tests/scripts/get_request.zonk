start {
	let response = read_string("https://postman-echo.com/get?arg1=hello");
	write_string("get_request_response.txt", response);
	println("Created file containing get response");
}
