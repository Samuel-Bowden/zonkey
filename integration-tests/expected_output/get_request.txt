Created file containing get response
